src/quacc/calculators/mrcc/io.py

```diff
@@ -0,0 +1,126 @@
+from __future__ import annotations
+
+from io import StringIO
+from pathlib import Path
+
+from ase.io import read
+from ase.units import Hartree
+from ase.utils import reader, writer
+
+# Made from NWChem interface
+
+
+@reader
+def read_geom_mrccinp(fd):
+    """Method to read geometry from an MRCC input file."""
+    lines = fd.readlines()
+    atomslength = 0
+    atomslenline = -1
+
+    # Find geometry region of input file.
+    for index, line in enumerate(lines):
+        if "xyz" in line:
+            startline = index + 3
+            atomslenline = index + 1
+        elif index == atomslenline:
+            atomslength = int(line)
+            break
+
+    # Format and send to read_xyz.
+    xyz_text = "%i\n" % (atomslength)
+    xyz_text += " geometry\n"
```

Comment:
Combine these operations in one line, e.g. `xyz_text =  f"{atomslength}\n geometry\n"